README.MD

```diff
@@ -23,160 +25,148 @@
 
-In order to run this project you need:
-
-> git clone https://github.com/DavidEspinoG/portfolio.git
-> (you can open it via live server in VS)
+You don't need any previous installation in order to run this project
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-> git clone https://github.com/DavidEspinoG/portfolio.git
+git clone https://github.com/DavidEspinoG/Awesome_books.git
 
 ### Install
 
-Install this project with:
-
-N/A
+This is not an install able project
 
 ### Usage
 
-To run the project just open it in VS and run liveServer
-
+There's no command required to run the project
 
 ### Run tests
 
-To run tests, run the following command:
+Linter Tests added ( npm run test )
 
-N/A
 ### Deployment
 
-You can deploy this project using:
-
-N/A
+This is not a deployable project
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a id="authors"></a>
 
+👤 Shayan
 
-👤 **David Espino**
+- GitHub: [@githubhandle](https://github.com/shayan1234554321)
+- Twitter: [@twitterhandle](https://twitter.com/shayan123455432)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shayan-khan20/)
+
+👤 Adil
+
+- GitHub: [@githubhandle](https://github.com/nxn404-n)
+- Twitter: [@twitterhandle](https://twitter.com/nxn_404)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nafeurrahmanadil/)
+
+👤 David Espino
 
 - GitHub: [@DavidEspinoG](https://github.com/DavidEspinoG)
 - Twitter: [@DavidEspinoG](https://twitter.com/DavidEspinoG)
 - LinkedIn: [davidespinog](https://linkedin.com/in/davidespinog)
 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a id="future-features"></a>
 
-
-- More projects are going to be added **[new_feature_1]**
-- A contact form
+> Would be adding contact page and css styling
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- CONTRIBUTING -->
 
 ## 🤝 Contributing <a id="contributing"></a>
 
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
+All issues are welcomed !! You can contribute by creating an issue.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a id="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project please star this repo.
+If you like this project please leave a ⭐️
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a id="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank microverse inc. for helping me in my professional carreer. 
+ Thanks to Microverse for project
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a id="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [REQUIRED] Kindly ensure that this project is properly licensed. Right now the license is broken. The license helps indicate that you are the owner of the project.
<img width="860" alt="Screenshot 2023-02-20 at 19 16 18" src="https://user-images.githubusercontent.com/58692590/220177147-72c4dc0d-8386-488b-9efb-35bf50bc5e42.png">